src/components/views/causes/create/CauseReviewStep.tsx

```
@@ -0,0 +1,816 @@
+					({selectedProjects?.length})
+				</InfoTitle>
+				<Row>
+					{selectedProjects &&
+						selectedProjects.map((project: any) => (
+							<Col lg={4} md={12} key={project.id}>
+								<CauseCreateProjectCard
+									project={project}
+									showOptions={false}
+								/>
+							</Col>
+						))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)